PureBasicIDE/SourceManagement.pb

```diff
@@ -2373,6 +2372,9 @@ Procedure RemoveSource(*Source.SourceFile = 0)
   ; There is almost no flicker anymore, so it actually looks quite good.
   FlushEvents()
   
+  ; Remove old EditorGadget
```

Comment:
Could you add a comment about why it needs to be at the very end of procedure, like:
"Fix a stack corruption on OSX, needs to be after FlushEvent()"
Because it's strange the order matters here. May be there is some event in the queue which still belongs to the editor gagdet, but it looks like a PB bug if it's the case.